deployment/buildAndDeploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ $TRAVIS_BRANCH == 'master' ]]; then
+  ./gradlew uploadArchives -PnexusUser=$NEXUS_USER -PnexusPassword=$NEXUS_PASSWORD -Psigning.keyId=$SIGNING_KEY_ID -Psigning.password=$SIGNING_PASSWORD -Psigning.secretKeyRingFile=../deployment/secret-keys-keanu.gpg
+else
+  ./gradlew check
```

Comment:
FYI I've changed this on another branch (building Python) so will change it here too once I merge yours into mine.

Reply:
Nice!